### ISSUE TITLE
AST names convert into links in chats 

### ISSUE THREAD

**Issue by kobakitty** [online-go/online-go.com#2144]
**Describe the bug**
When writing "tournament #number" in chat its automatically converted into a link by ogs. That would be great on itself, but the names of automatic sitewide tournaments have the same format so when copypasteing tournament name into chat there will be weird link for the tournament which have an ID of tournaments year 

**To Reproduce**
Copypaste name of AST in chat and see what happens xD

Maybe either have the copypasted tournament name automatically convert into link for the tournament it actually is (might be bad idea considering how many custom tournaments have had the name "test" or some else commonly used word), OR tweak the AST's names so that the middle part of it isnt converted into a hyperlink.

**Screenshots**
<img width="653" alt="Screenshot 2023-02-05 at 21 04 50" src="https://user-images.githubusercontent.com/29932807/216839819-c42fac88-f8d4-408e-8780-c5afbe3f9ec2.png">
The link the in middle is for tournament id 2023, which was played 10 years ago, seems very confusing ^^

**Comment by benjaminpjones** [comment]
Good find! What does AST stand for BTW?  I've only seen that used for "Abstract Syntax Tree"...

**Comment by BHydden** [comment]
@benjaminpjones did you mean to close this as completed before submitting a PR to fix it? 🤔

**Comment by benjaminpjones** [comment]
Oops fat fingered the comment button

**Comment by BHydden** [comment]
Thought that may have been the case but didn't want to necro it just in case 😅

**Comment by benjaminpjones** [comment]
Marking as "good first issue" because it's easy to test out code in chat_markup.tsx.

Code point for the regex that (I think) triggers this: https://github.com/online-go/online-go.com/blob/42d9957567628aad9471fff81f654404e7fa9671/src/components/Chat/chat_markup.tsx#L287-L296

I think the hard part is what to change the regex to (if that is the route we want to take)

**Comment by mister-dl** [comment]
Yup I was able to repro this by pasting " Live 9x9 Double Elimination Tournament 2025-02-13 14:30 " into the chat.

![Image](https://github.com/user-attachments/assets/14fc8d54-d4f1-466d-ba76-6f8d390ef4a0)

I'll take a stab at fixing this
